diff --git a/node_modules/@expo/vector-icons/build/createIconSet.d.ts b/node_modules/@expo/vector-icons/build/createIconSet.d.ts
index 3af473b..adf5fbe 100644
--- a/node_modules/@expo/vector-icons/build/createIconSet.d.ts
+++ b/node_modules/@expo/vector-icons/build/createIconSet.d.ts
@@ -1,5 +1,6 @@
 import React from 'react';
-export { DEFAULT_ICON_COLOR, DEFAULT_ICON_SIZE } from './vendor/react-native-vector-icons/lib/create-icon-set';
+export const DEFAULT_ICON_COLOR: string;
+export const DEFAULT_ICON_SIZE: string;
 export default function (glyphMap: any, fontName: any, expoAssetId: any, fontStyle?: any): {
     new (props: Readonly<{}>): {
         _mounted: boolean;
